src/chemistry/Chemistry.hpp

```diff
@@ -90,6 +94,9 @@ template <typename problem_t> auto computeChemistry(amrex::MultiFab &mf, const R
 			// call the EOS to set initial internal energy e
 			eos(eos_input_re, chemstate);
 
+			// set initial Tdust to CMB
+			chemstate.aux[0] = TCMB;
```

Comment:
Is `chemstate.aux[0]` the dust temperature for all chemistry networks? We need to document this somewhere if so.